File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/exception/MappingException.java
package no.nav.melosys.eessi.models.exception;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@ResponseStatus(value = HttpStatus.BAD_REQUEST)
public class MappingException extends Exception {

    public MappingException(String message) {
        super(message);
    }

    public MappingException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/exception/MappingException.kt
package no.nav.melosys.eessi.models.exception;

public class MappingException extends RuntimeException {

    public MappingException(String message) {
        super(message);
    }

    public MappingException() {
        super();
    }

    public MappingException(String message, Throwable cause) {
        super(message, cause);
    }
}
